Assets/Plugins/iOS/Shopify/Checkout/Checkout.WebViewController.swift

```diff
@@ -44,9 +48,15 @@ class WebViewController: UIViewController {
     }
     
     init(url: URL, delegate: WebCheckoutDelegate) {
-        self.url = url
+        self.startURL = url
+        self.lastGoodURL = url
         self.delegate = delegate
-        
+
+        let errorPageFileURL = Bundle.main.url(forResource: "errorPage", withExtension: "html")!
+        let polarisCSSFileURL = Bundle.main.url(forResource: "polaris-1.0.2", withExtension: "css")!
```

Comment:
The `main` bundle what you want here? I'm not sure what context this runs in but if this bundle is a dependency of an application bundle, your resources will come back `nil`. It's entirely possible that this will never be the case but it may be safer to use `Bundle(for: WebViewController.self)` instead. Your call.